### PR TITLE
[Release] 2022-08-16

### PR DESCRIPTION
**Pull request by rossifrancesco**
Plugin version: 0.11
Grafana version: 8.5.5

### CODE REVIEW

**Review by crohn — APPROVED** [review]
LGTM

**Comment by crohn** [comment]
Before merging, let's wait for @hyperboreans confirmation on [575341f](https://github.com/draios/grafana-sysdig-datasource/pull/94/commits/575341fe521fd2f4c75c6c575bdfbb73e2d19778).

File: README.md
Comment (hyperboreans) [inline README.md:13]: 8.5.5. right?

**Review by hyperboreans — APPROVED** [review]

**Pull request merged**
